### PR TITLE
Stellarium 24.3-2

### PR DESCRIPTION
**Pull request by alex-w**
Switch to use system-provided QXlsx library

### CODE REVIEW

File: sci-astronomy/stellarium/stellarium-24.3.recipe [hunk 1]
```diff
@@ -35,6 +35,7 @@ REQUIRES="
 	lib:libQt6SerialPort$secondaryArchSuffix
 	lib:libQt6Widgets$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
+	lib:libQXlsxQt6$secondaryArchSuffix
```
Comment (jmairboeck) [inline sci-astronomy/stellarium/stellarium-24.3.recipe:38]: This should be sorted alphabetically (i.e. before libz).

File: sci-astronomy/stellarium/stellarium-24.3.recipe [hunk 2]
```diff
@@ -52,6 +53,8 @@ BUILD_REQUIRES="
 	devel:libQt6SerialPort$secondaryArchSuffix
 	devel:libQt6Widgets$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
+	qxlsx_devel$secondaryArchSuffix
```
Comment (jmairboeck) [inline sci-astronomy/stellarium/stellarium-24.3.recipe:56]:
```suggestion
	devel:libQXlsxQt6$secondaryArchSuffix
```
and sort alphabetically.

File: sci-astronomy/stellarium/stellarium-24.3.recipe [hunk 3]
```diff
@@ -52,6 +53,8 @@ BUILD_REQUIRES="
...
+	qxlsx_qt6$secondaryArchSuffix
```
Comment (jmairboeck) [inline sci-astronomy/stellarium/stellarium-24.3.recipe:57]: Ideally, this shouldn't be required, but qxlsx_devel is missing a requirement on qxlsx_qt6, it seems ...

**Review by jmairboeck — CHANGES_REQUESTED** [review]

File: sci-astronomy/stellarium/stellarium-24.3.recipe (hunk 3 above)
Comment (jmairboeck) [reply sci-astronomy/stellarium/stellarium-24.3.recipe:57]: I fixed this in 82168a5, so this line can be removed after rebasing this PR.

**Comment by alex-w** [comment]
Done

**Review by jmairboeck — APPROVED** [review]

**Comment by jmairboeck** [comment]
I can't really test it right now, because today's update broke haikuporter (and the shell in general somewhat; I'll have to investigate that), but the changes LGTM.

**Pull request merged**